geopandas/tests/test_overlay.py

```diff
@@ -86,6 +86,31 @@ def test_duplicate_column_name(self):
         df = overlay(self.polydf, polydf2r, how="union")
         self.assertTrue('Shape_Area_2' in df.columns and 'Shape_Area' in df.columns)
 
+    def test_geometry_not_named_geometry(self):
+        # Issue #306
+        df = overlay(self.polydf, self.polydf2, how="union")
+        self.assertEquals(df.shape, self.union_shape)
```

Comment:
This test is redundant, it should already be well tested in the other test cases. I'd omit this.